gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -410,13 +399,18 @@ gst_tensor_filter_reset (GstTensorFilter * self)
   prop->custom_properties = NULL;
 
   prop->input_configured = FALSE;
-  gst_tensor_filter_deallocate_tensor_name (&prop->input_meta);
+  gst_tensors_info_free (&prop->input_meta);
+  gst_tensors_info_init (&prop->input_meta);
 
   prop->output_configured = FALSE;
-  gst_tensor_filter_deallocate_tensor_name (&prop->output_meta);
+  gst_tensors_info_free (&prop->output_meta);
```

Comment:
`gst_tensor_info_init` sets `info->name` to `NULL`. (it does without checking it allocation, so it assumes that info->name has been freed.)
So, cant we call `gst_tensor_info_free` for each tensor inside `gst_tensor_info_init` rather than calling here separately.

Reply:
I removed _reset() function. When the vmethod _finalize() called, we don't need to initialize the variables.